assignments/callbacks.js

```diff
@@ -24,24 +27,34 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items,callback);
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
+last(items,callback);
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x+y);
 }
+sumNums(3,9,callback);
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x*y);
 }
+multiplyNums(1,3,callback);
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  return (list.find(e => (e == item))) ? cb(true) : cb(false);
```

Comment:
good job using `.find()`, it's a pretty powerful method. it works well here, but if you want to use an even simpler method try `.includes()`. it would save you a little bit of typing and looks slightly cleaner. very good job with the ES6 syntax and ternary though, it all looks very clean in one line